src/css/cart.css

```diff
@@ -0,0 +1,14 @@
+@import "../common.css";
+
+.cart-container{
+    margin-top: 8rem;
+}
+
+.card-items-qty-input{
+    width: 2rem;
```

Comment:
can give max or min-width for responsiveness, can add later on:)

Reply:
Yes, will take this into account.